composer.json

```
@@ -57,7 +57,7 @@
         "prestashop/blockreassurance": "^5.1",
         "prestashop/blockwishlist": "^2.0",
         "prestashop/circuit-breaker": "^4.0",
-        "prestashop/classic": "^2.0",
+        "prestashop/classic": ">=2.0,<2.1",
```

```suggestion
        "prestashop/classic": "~2.0.0",
```
i think it's the same? or `2.0.*`? :D

---

it was the only syntax I was sure about by memory :D 
I'll check yours and update :+1: